app/containers/MessageBox/RightButtons.android.js

```diff
@@ -1,31 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { SendButton, AudioButton, ActionsButton } from './buttons';
+import { SendButton, ActionsButton } from './buttons';
 
 const RightButtons = React.memo(({
-	theme, showSend, submit, recordAudioMessage, recordAudioMessageEnabled, showMessageBoxActions
+	theme, showSend, submit, showMessageBoxActions
 }) => {
 	if (showSend) {
 		return <SendButton onPress={submit} theme={theme} />;
 	}
-	if (recordAudioMessageEnabled) {
-		return (
-			<>
-				<AudioButton onPress={recordAudioMessage} theme={theme} />
-				<ActionsButton onPress={showMessageBoxActions} theme={theme} />
-			</>
-		);
-	}
```

Comment:
Why not just change here to something like:
```
if (showSend) {
  return <SendButton />;
}
return (
  <>
    {!recording ? <ActionsButton /> : null}
    <AudioButton />
  </>
)
```

Reply:
One of the main reasons was to not pass the `recordingCallback` and the `recording` states through the `RightButtons` component.  Also, `finishAudioMessage` will have to be passed twice

Reply:
ok, you can get on with your logic.